kit-slides.typ

```diff
@@ -259,3 +261,69 @@
 
     slide(title: title, body)
 }
+
+#let roundedblock(title: [], body, color: white, title-color: black) = {
+  block(width: 100%, fill: color.lighten(85%), inset: 0pt, clip: true, radius: (
+    "bottom-left": 9pt,
+    "top-right": 9pt,
+  ))[
+    #block(
+      width: 100%, 
+      inset: (left: 0.5em, top: 0.3em, bottom: 0.4em), 
+      fill: gradient.linear(
+        (color, 0%), (color, 87%), (color.lighten(85%), 100%),
+        dir: ttb
+      ), 
+      radius: (
+        "top-right": 9pt, // Todo: Remove for typst 0.9.0
+      ),
+      text(fill: title-color, title)
```

Comment:
Instead of taking an extra title-color argument and coloring the title here you can just pass a already colored content to the title argument.